### PR TITLE
Uniformize tests with the new API

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#716

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 1.2.0 - [#18](https://github.com/openfisca/extension-template/pull/18)
+
+* Technical change
+* Details:
+  - Adapt to Core v25
```
Comment (bonjourmauko) [inline CHANGELOG.md]: Add example as in core and country template? (note this is a question, not a request for change).

**Review by bonjourmauko — CHANGES_REQUESTED** [review]
Same as in openfisca/country-template#66: from a user's perspective, it is a breaking change IMHO.

**Comment by fpagnoux** [comment]
Could you elaborate on why this is a breaking change? What use-case are you thinking about?

I guess we could consider that a BC if we consider that users can run tests of their own that are not part of the package 🤔

**Comment by fpagnoux** [comment]
Would you have time to suggest or add a few commits to make this PR acceptable?

**Comment by bonjourmauko** [comment]
Hi @fpagnoux, I was thinking by the contributor point of view, but I'm now convinced it is a minor. Go !

**Review by bonjourmauko — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
         ("share/openfisca/openfisca-extension-template", ["CHANGELOG.md", "LICENSE", "README.md"]),
         ],
     install_requires = [
-        "OpenFisca-Country-Template >= 3,  < 4",
+        "OpenFisca-Country-Template >= 3.6.0.rc0,  < 4",
```
Comment (bonjourmauko) [inline setup.py]: Note: do not forget to remove the release candidate version.

**Pull request merged**